src/util/location.ts

```diff
@@ -9,11 +9,7 @@ export const generateLink = (
   let path: string;
 
   if (target !== def) {
-    if (current === def) {
-      path = `/${target}/`;
-    } else {
-      path = location.pathname.replace(current, target);
-    }
+    path = `/${target}/`;
```

Comment:
It's a great idea! Why have I unnecessarily complicated the code here? haha 🤷‍♂️ 
All existing tests still pass, and that seems fine to me. Thank you for the work!

Reply:
Oh hey, shall I also remove 'current' arg from the generateLink function, since it's no longer needed?

Reply:
Yes, you've done a great job, thanks for removing the unnecessary parameter!